build.sbt

```diff
@@ -8,6 +8,7 @@ scalaVersion := "2.11.7"
 
 libraryDependencies ++= Seq(
   "com.softwaremill.reactivekafka" %% "reactive-kafka-core" % "0.8.1",
+  "com.typesafe.akka" %% "akka-stream-experimental" % "2.0-M1",
```

Comment:
akka-stream-experimental:2.0-M2 should pull in as part of reactive-kafka-core
Can you make changes to reactive-kafka-core so it pulls in the latest akka-stream?